test/test_jit.py

```diff
@@ -1787,6 +1787,36 @@ def forward(self, x):
         res = get_forward(m._c)(x)
         self.assertEqual(res, ref_res)
 
+    def test_swap_dequantize(self):
+        class M(torch.nn.Module):
+            def __init__(self):
+                super(M, self).__init__()
+                self.maxpool = torch.nn.MaxPool2d(kernel_size=3)
+                self.avgpool = torch.nn.AdaptiveAvgPool2d((1, 1))
+
+            def forward(self, x):
+                x = torch.dequantize(x)
```

Comment:
We should check over all the ops that we support here.

Reply:
will add a separate test for checking all supported ops, that does not run the model so we don't need to think about how to connect all supported ops.
This test is also checking that results are matching.